pkg/network/render.go

```diff
@@ -38,6 +38,12 @@ var dualStackPlatforms = sets.NewString(
 	string(configv1.OpenStackPlatformType),
 )
 
+var conversionToDualStackPlatforms = sets.NewString(
+	string(configv1.BareMetalPlatformType),
+	string(configv1.NonePlatformType),
+	string(configv1.VSpherePlatformType),
+)
```

Comment:
question: Previously, only `OpenStack` is rejected. Just wondering if we confirmed this is OK?

Reply:
This is the "allow" list so it contains the list of platforms that allow conversion to DualStack on Day-2. The code earlier was looking at OpenStack platform as the reject list. It could be implemented either ways.

Reply:
Oh sounds good! I was just unsure if there are any other platforms that allow dualstack, which is not yet accounted for here...